.gitignore

```diff
@@ -1,4 +1,6 @@
 .bitrise*
 
 bitrise-add-new-project
-docker_linux_interactive/bitrise-add-new-project
\ No newline at end of file
+docker_linux_interactive/bitrise-add-new-project
+
+.idea
```

Comment:
Small thing, but let's be a bit more concrete here: `/.idea/` (`.idea` directory, not file, in the root of the repo).

I don't anticipate any issues out of this, but this pattern matches _anything_ (file or dir) _anywhere_ in the repository that's called `.idea`.